frontend/src/components/NavigateButton/styles.module.scss

```
@@ -0,0 +1,55 @@
+@import "src/components/styles/variables/color";
+@import "src/components/styles/variables/font";
+@import "src/components/styles/variables/property";
+
+.activSwitcher {
+  display: flex;
+  width: 50%;
+  background-color: $activ-button;
+  border-radius: $small-border-radius;
+  border: 0;
+  box-shadow: $shadow;
+}
+
+.switcher {
+  display: flex;
+  width: 50%;
+  border-radius: $small-border-radius;
+  border: 0;
+}
+
+.default {
+  display: flex;
+  background-color: $default-button-color;
+  border: 0;
+}
+
+.default:hover {
+  display: flex;
+  background-color: $default-button-color;
+  border: 0;
+}
+
+.main {
+  display: flex;
+  border-color: $white-border;
+}
+
+.main:hover {
+  display: flex;
+  border-color: $activ-button;
+}
```

Please, could you use SCSS selectors

https://sass-lang.com/documentation/style-rules/parent-selector/

```suggestion
.main {
  display: flex;
  border-color: $white-border;

  &:hover {
    display: flex;
    border-color: $activ-button;
}
```